packages/oidc-mock-provider/bin/oidc-mock-provider.js

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env node
+'use strict';
+
+const { OIDCMockProvider } = require('..');
+
+const DEFAULT_PORT = 28200;
+const DEFAULT_HOST = 'localhost';
+
+const argv = require('yargs')
+  .option('port', {
+    alias: 'p',
+    type: 'string',
+    desc: 'Port to run the server on. Defaults to 0 (auto-assigns to a random port).',
+  })
+  .option('host', {
+    alias: 'h',
+    type: 'string',
+    desc: 'Hostname for the server to listen upon. Defaults to localhost.',
+  })
+  .example(
+    '$0 -p 28200',
+    'Start the OIDC mock identity provider server on port 28200'
+  )
+  .help().argv;
+
+const DEFAULT_TOKEN_PAYLOAD = {
+  expires_in: 3600,
+  payload: {
+    // Define the user information stored inside the access tokens.
+    groups: ['testgroup'],
+    sub: 'testuser',
+    aud: 'resource-server-audience-value',
+  },
+};
+
+(async function main() {
+  const oidcMockProviderConfig = {
+    getTokenPayload() {
+      return DEFAULT_TOKEN_PAYLOAD;
+    },
+    overrideRequestHandler() {
+      return () => {};
+    },
```

Comment:
```suggestion
```

(This is optional, and it doesn’t really matter that much either way, but it doesn’t return a function so I was going to leave a suggestion anyway :slightly_smiling_face:)